### PR TITLE
Misc Fixes

### PR DESCRIPTION
**Pull request by artdude543**
- Fixed: Example code for joining chat.
- Changed: Used faster for method to loading the services.
- Tweaked: Some formatting and spelling issues.

### CODE REVIEW

File: lib/providers/oauth.js [hunk 1]
```diff
@@ -141,7 +141,7 @@ OAuthProvider.prototype.attempt = Bluebird.method(function (redirect, qs) {
     if (!qs.code) {
         throw new errors.AuthenticationFailedError('No error was given, ' +
             'but a code was not present in the query string. Make sure ' +
-            'you\'re using the oauth client correctly.'); // silly devlopers
+            'you\'re using the oauth client correctly.'); // silly developers
```
Comment (connor4312) [inline lib/providers/oauth.js]: [no](https://twitter.com/iamdevloper)

File: lib/client.js [hunk 2]
```diff
@@ -19,8 +17,8 @@ function Client () {
         public: 'https://beam.pro'
     };
 
-    for (var key in Services) {
-        this[key] = new Services[key](this);
+    for (var keys = Object.keys(Services), i = 0, length = keys.length; i < length; i++) {
+        this[keys[i]] = new Services[keys[i]](this);
     }
```
Comment (connor4312) [inline lib/client.js]: If you want to avoid prototype properties, do `Object.keys(Services).forEach(function (key) {`

Comment (connor4312) [inline lib/client.js]: (moved my comment on this to the commit)

**Comment by artdude543** [comment]
Someone else can make a PR to fix things if they want. Not going to sit here and argue with whats standard or not. Or about linting methods.

**Comment by connor4312** [comment]
Feedback and discussion half the purpose of pull requests, it's par for the course when contributing to any projects! Our own PRs internally go through rigorous review and are rarely accepted without changes. I apologize if you felt that the comments were meant to be derogatory or inflammatory; that wasn't my intention, nor--I think I can safely say--that of any of the others who also commented on this request.

**Comment by JamyDev** [comment]
:( @artdude543

**Comment by rfox90** [comment]
Not my Intention either. I was actually interested in your reasoning.

Pull requests often end up with weeks of intellectual discussion.

**Comment by MCProhosting-Matt** [comment]
:( this makes me sad

**Pull request closed without merging**